hi-ml-azure/src/health_azure/himl.py

```diff
@@ -419,6 +419,7 @@ def submit_run_v2(
     pytorch_processes_per_node: Optional[int] = None,
     use_mpi_run_for_single_node_jobs: bool = True,
     display_name: Optional[str] = None,
+    argument_prefix: str = "--",
```

Comment:
```suggestion
    hyperdrive_argument_prefix: str = "--",
```

Reply:
Thanks! Fixed in [608d7ae](https://github.com/microsoft/hi-ml/pull/936/commits/608d7ae4ea7f018be79b16c99c2208849c27242a).